.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Problem or regression with an existing feature
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Before opening a bug report, please search open and closed issues to see if the problem has already been addressed. If there is such a report, and your issue is not adequately addressed there, please reference the issues in your new report.
+
+## Bug Report
+
+Please pick one of the following categories:
+
+### Build Problem
+
+Please provide your complete command line invocation, and attach a log of the console output.
+
+### Incorrect Functionality and General Questions
+
+Describe the issue here.
+
+## To Reproduce
+
+1. Operating System
+2. Python version
+3. Example snippet that demonstrates the issue - if it's a build issue,
+4. OpenTimelineIO release version or commit hash
+5. Compiler information:
+    on Mac, type `clang -v`, and paste the results here
+    on Linux, type `gcc -v`, and paste the results here
+    on Windows, type `cl`, and paste the results here.
+    If you are unable to determine your compiler via these commands, please indicate that here.
```

Comment:
I don't think it's relevant here since `pyaaf2` is pure Python? Or potentially we keep it here just in case someone simply installs `otio-aaf-adapter` without actually installing OTIO (which would mean OTIO would be installed by `otio-aaf-adapter`)?

Reply:
I can get rid of this, I just missed it when copying the file over from the main OTIO repo